### ISSUE TITLE
Remove AI inventory

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/cto-website#1026]

**Comment by jfredrickson** [comment]
Want a redirect from the old URL to the new one, or a message on the page about the new location, or just plain old delete the page?

**Comment by vickimcfadden** [comment]
Good idea, let's put in a redirect to the new page, at least for a few months.
